contracts/SSVNetwork.sol

```diff
@@ -142,91 +142,27 @@ contract SSVNetwork is UUPSUpgradeable, OwnableUpgradeable, ISSVNetwork {
         emit OperatorAdded(id, msg.sender, publicKey, fee);
     }
 
-    function removeOperator(uint64 id) external override {
-        Operator memory operator = operators[id];
-        if (operator.owner != msg.sender) revert CallerNotOwner();
-
-        operator.getSnapshot();
-        uint64 currentBalance = operator.snapshot.balance;
-
-        operator.snapshot.block = 0;
-        operator.snapshot.balance = 0;
-        operator.validatorCount = 0;
-        operator.fee = 0;
-
-        operators[id] = operator;
-
-        if (currentBalance > 0) {
-            _transferOperatorBalanceUnsafe(id, currentBalance.expand());
-        }
-        emit OperatorRemoved(id);
+    function removeOperator(uint64 operatorId) external override  {
+        _removeOperator(operatorId, operators[operatorId]);   
     }
 
     function declareOperatorFee(
         uint64 operatorId,
         uint256 fee
-    ) external override onlyOperatorOwnerOrContractOwner(operatorId) {
-        if (fee < MINIMAL_OPERATOR_FEE) revert FeeTooLow();
-
-        uint64 shrunkFee = fee.shrink();
-
-        // @dev 100%  =  10000, 10% = 1000 - using 10000 to represent 2 digit precision
-        uint64 maxAllowedFee = (operators[operatorId].fee *
-            (10000 + operatorMaxFeeIncrease)) / 10000;
-
-        if (shrunkFee > maxAllowedFee) revert FeeExceedsIncreaseLimit();
-
-        operatorFeeChangeRequests[operatorId] = OperatorFeeChangeRequest(
-            shrunkFee,
-            uint64(block.timestamp) + declareOperatorFeePeriod,
-            uint64(block.timestamp) +
-                declareOperatorFeePeriod +
-                executeOperatorFeePeriod
-        );
-        emit OperatorFeeDeclared(msg.sender, operatorId, block.number, fee);
+    ) external override  {
+        _declareOperatorFee(operatorId, operators[operatorId], fee);
```

Comment:
@mtabasco lets add a comment stating that 0 fee can't be changed and usually should be used with private operators as well